config/rbac/role.yaml

```diff
@@ -14,6 +14,12 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
```

Comment:
In case we decide we want to validate against nodes, we'll have to add this to the charts too.

Reply:
yes, I saw the tests failing due to this

Reply:
I will create a PR on the operator for the same